include/clients/rt/LoudnessClient.hpp

```diff
@@ -122,6 +133,8 @@ class LoudnessClient : public FluidBaseClient, public AudioIn, public ControlOut
   algorithm::Loudness                                mAlgorithm;
   BufferedProcess                                    mBufferedProcess;
   FluidTensor<double, 1>                             mDescriptors;
+
+  index mMaxOutputSize;
```

Comment:
this is always 2, so no need for a variable

Reply:
Okay cool! Less code is good :)

Reply:
Resolved in b75b213